charts/apps/immich/values.yaml

```diff
@@ -99,19 +99,20 @@ machine_learning:
           port: 3003
           protocol: HTTP
 
-  command: "/bin/sh"
+  command: "python"
   args:
-    - "./entrypoint.sh"
+    - "src/main.py"
   env:
     NODE_ENV: "production"
+    TRANSFORMERS_CACHE: /usr/src/app/.transformers_cache
```

Comment:
The container already sets the cache to `/cache`, should we just use that?

Reply:
generally, we need to make sure the directory is writable to the container. I'd rather not use a top-level directory like `/cache` as it would need root permissions on most distros. I moved the folder to the rest of the application in `/usr/src/app`  so it's easier to persist if need be.

Reply:
Sure 👍